<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Geng_gai</class>
 <widget class="QWidget" name="Geng_gai">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Geng_gai</string>
  </property>
  <widget class="QLineEdit" name="score_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>333</x>
     <y>302</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>306</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>课名4：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>118</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="score_5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>333</x>
     <y>352</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>206</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>成绩：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>406</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>成绩：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>206</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>课名2：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>256</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>成绩：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>118</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>66</y>
     <width>1</width>
     <height>500</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Kao_he/shen.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="course_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>252</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="score_6">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>333</x>
     <y>402</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>256</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>课名3：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="course_6">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>402</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>306</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>成绩：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>506</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>成绩：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="score_8">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>333</x>
     <y>502</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>95</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListView" name="student_ti">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>261</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Kao_he/shen.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>156</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>成绩：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>356</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>成绩：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="score_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>333</x>
     <y>202</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="course_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>202</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="course_5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>352</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="course_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>302</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>456</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>课名7：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>156</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>课名1：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="score_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>333</x>
     <y>252</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>506</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>课名8：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="course_8">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>502</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>456</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>成绩：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="course_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>152</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="score_7">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>333</x>
     <y>452</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>60</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>信    息    提    示</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="tian_jia_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>546</y>
     <width>311</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Kao_he/shen.png);</string>
   </property>
   <property name="text">
    <string>成  绩  更  改  完  成</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>356</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>课名5：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="course_7">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>452</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>95</y>
     <width>358</width>
     <height>1</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Kao_he/shen.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>146</y>
     <width>1</width>
     <height>420</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Kao_he/shen.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="score_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>333</x>
     <y>152</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>406</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>课名6：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>95</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>学号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>600</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_2">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>60</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>1122334455</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>64</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>学号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tian_jia_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Kao_he/shen.png);</string>
   </property>
   <property name="text">
    <string>查 询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>550</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>别 忘 了 去 导 出 保 存 嗷 🌷 ~</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="qie_mo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Kao_he/shen.png);</string>
   </property>
   <property name="text">
    <string>模式切换</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="age_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>118</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>95</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>年龄</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="gender_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>118</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>95</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label_21</zorder>
  <zorder>score_4</zorder>
  <zorder>label_15</zorder>
  <zorder>name_1</zorder>
  <zorder>score_5</zorder>
  <zorder>label_4</zorder>
  <zorder>label_8</zorder>
  <zorder>label_17</zorder>
  <zorder>label_5</zorder>
  <zorder>id_1</zorder>
  <zorder>label_22</zorder>
  <zorder>course_3</zorder>
  <zorder>score_6</zorder>
  <zorder>label_18</zorder>
  <zorder>course_6</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>score_8</zorder>
  <zorder>label</zorder>
  <zorder>student_ti</zorder>
  <zorder>label_3</zorder>
  <zorder>label_10</zorder>
  <zorder>score_2</zorder>
  <zorder>course_2</zorder>
  <zorder>course_5</zorder>
  <zorder>course_4</zorder>
  <zorder>label_12</zorder>
  <zorder>label_19</zorder>
  <zorder>score_3</zorder>
  <zorder>label_13</zorder>
  <zorder>course_8</zorder>
  <zorder>label_9</zorder>
  <zorder>course_1</zorder>
  <zorder>score_7</zorder>
  <zorder>label_20</zorder>
  <zorder>tian_jia_2</zorder>
  <zorder>label_16</zorder>
  <zorder>course_7</zorder>
  <zorder>label_24</zorder>
  <zorder>label_23</zorder>
  <zorder>score_1</zorder>
  <zorder>label_14</zorder>
  <zorder>label_2</zorder>
  <zorder>id_2</zorder>
  <zorder>label_25</zorder>
  <zorder>tian_jia_3</zorder>
  <zorder>label_26</zorder>
  <zorder>qie_mo</zorder>
  <zorder>age_1</zorder>
  <zorder>label_11</zorder>
  <zorder>gender_1</zorder>
  <zorder>label_27</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>tian_jia_2</sender>
   <signal>clicked()</signal>
   <receiver>Geng_gai</receiver>
   <slot>geng_gai()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>605</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tian_jia_3</sender>
   <signal>clicked()</signal>
   <receiver>Geng_gai</receiver>
   <slot>cha_stu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>-5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qie_mo</sender>
   <signal>clicked()</signal>
   <receiver>Geng_gai</receiver>
   <slot>qie_qie()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>geng_gai()</slot>
  <slot>cha_stu()</slot>
  <slot>qie_qie()</slot>
 </slots>
</ui>
